### PR TITLE
Fix spelling/name of OSTree

### PR DESCRIPTION
**Pull request by VladimirSlavik**
It's spelled that way everywhere. I think it might be better to stick to that?

See https://translate.fedoraproject.org/translate/anaconda/master/en/?checksum=e440ad521e7a3ffe

### CODE REVIEW

**Review by martinpitt — APPROVED** [review]
Thanks! That is the official spelling indeed.

**Review by poncovka — APPROVED** [review]
Thank you! I think that we have some tests that checks for the bad strings. Could you extend them, please?

**Comment by VladimirSlavik** [comment]
Thanks for the idea and pointers, done!

File: tests/gettext_tests/style_guide.py [hunk 1]
```diff
@@ -31,6 +31,7 @@
                '[Cc]an not':     'cannot',
                '(?i)mountpoint': 'mount point',
                'Ok':             'OK',
+               '(?i)os tree ':   'OSTree',
```
Comment (martinpitt) [inline tests/gettext_tests/style_guide.py:34]: 💿🌲 clearly! :wink:

**Review by martinpitt — APPROVED** [review]
Cheers!

**Review by poncovka — APPROVED** [review]
Thanks!

**Pull request merged**